workflow/envs/samtools_stats.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - conda-forge
+  - bioconda
+  - defaults
+dependencies:
+  - samtools==1.10
```

Comment:
our container has version 1.14, should we change to same here?
```suggestion
  - samtools==1.14
```

Reply:
It is from the wrapper but I guess it should be ok to change